petclinic-frontend/src/shared/models/path.routes.ts

```diff
@@ -1,6 +1,7 @@
 export enum AppRoutePaths {
   Default = '/',
   Inventories = '/inventories',
+  Vet = '/vet',
```

Comment:
For next time it is always best to append new items at the bottom of a list. If you add it in the middle all the lines are shifted down and the risk of a merge conflict is much greater.

Reply:
Ok thanks for the tip.